### PR TITLE
4989-Account_Overview_Edit: Edit account details form.

### PR DESCRIPTION
**Pull request by dfennessy**
**What this PR does / why we need it**:

> Lists UI Strings for Account settings > Overview > Edit account details

**Which issue(s) this PR fixes** 

> `fixes #<4989>`

**Verification steps** 

> None

**Special notes for your reviewer**:

- UI Strings Jira: [THREESCALE-4989](https://issues.redhat.com/browse/THREESCALE-4989)
- Most of the strings contain text similar to the mockup.
- For strings different to the mockup, you will see this tag: [PROPOSED]

### CODE REVIEW

**Comment by dfennessy** [comment]
I removed the strings for default values. How do we represent the placeholder strings? @porueesq

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+page_title: Edit account details | Portafly
+page_title_description: Page title of the subsection for Edit account details
+body_title: Edit account details
+body_title_description: Body title of the page for Edit account details
+
+# Main area
+
+organization_name_label: Organization name
+organization_name_label_description: Label for the field that holds the name of the organization that owns the account.
+organization_name_validation: {{organization_name_validation}} cannot be blank. Please add your {{organization_name_validation}}.
```
Comment (porueesq) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:11]:
```suggestion
organization_name_validation: "<i>Organization name</i> cannot be blank. Please add your organization name."
```

Comment (porueesq) [reply portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:11]: Let's use:
* Italics when referring to field names > _Organization name_ cannot be blank.
* Bold letters when referring to buttons or actions > Choose an item to enable the **Delete** button.
* Italic bold when referring to user names. > Hello **_John_**, You have 5 new messages.

WDYT @dfennessy @cntlsn

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+country_selectlist_label: Country
+country_selectlist_label_description: Label for the field that holds the name of the organization's country from the select list.
+
+postal_code_label: Postal code
+postal_code_label_description: Label for the field that holds the postal code of the organization.
+
+vat_code_label: VAT identification number
+vat_code_label_description: Label for the field that holds the VAT identification number of the organisation.
+
+time_zone_selectlist_label: Time zone
```
Comment (porueesq) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:37]:
```suggestion
time_zone_selectlist: Time zone
```

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+time_zone_selectlist_label_description: Label for the field that holds the time zone of the organization from the select list.
```
Comment (porueesq) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:38]:
```suggestion
time_zone_selectlist_description: Label for the field that holds the time zone of the organization from the select list.
```

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+apartment_suite_helper: Examples: Apartment No.4, 3A, 8-1, 2nd 3rd
+apartment_suite_helper_description: This is the helper text that indicates to the user how to fill in the apartment/suite field.
+
+city_label: City
+city_label_description: Label for the field that holds the name of the city where the organization is located.
+
+state_region_label: State, region, province
+state_region_label_description: Label for the field that holds the name of the state, region, province where the organization is located.
+
+country_selectlist_label: Country
```
Comment (porueesq) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:28]:
```suggestion
country_selectlist: Country
```

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+country_selectlist_label_description: Label for the field that holds the name of the organization's country from the select list.
```
Comment (porueesq) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:29]:
```suggestion
country_selectlist_description: Label for the field that holds the name of the organization's country from the select list.
```

**Comment by porueesq** [comment]
@dfennessy Some suggestions, and questions for your consideration. See that I've suggested removing the `_label` from some elements, like `selectlist`, for the purpose of simplification.

File: portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml (hunk 1 above)
Comment (josemigallas) [inline portafly/src/i18n/locales/en/account_settings/overview/edit_account_details.yml:11]: This phrase has to be between quotes.

**Review by josemigallas — CHANGES_REQUESTED** [review]

**Review by josemigallas — APPROVED** [review]

**Review by cntlsn — APPROVED** [review]

**Pull request merged**